builders/build/runtime-upgrades.md

```diff
@@ -80,3 +80,4 @@ Not all runtime upgrades are released on each network, as sometimes after releas
 |                                           2902                                           |                                  -                                   |                                   -                                   | [6732678](https://moonbase.subscan.io/block/6732678){target=\_blank} |
 | [3000](https://forum.moonbeam.network/t/runtime-rt3000-schedule/1752/2){target=\_blank}  |                                  -                                   | [7043011](https://moonriver.subscan.io/block/7043011){target=\_blank} | [7299818](https://moonbase.subscan.io/block/7299818){target=\_blank} |
 |                                           3001                                           | [6593037](https://moonbeam.subscan.io/block/6593037){target=\_blank} |                                   -                                   |                                  -                                   |
+|                                           3100                                           |                                    -                                    |                                   -                                   |[8034666](https://moonbase.subscan.io/block/8034666){target=\_blank}|
```

Comment:
When there is a a major runtime bump (i.e., from 3000 to 3100), there should always be a forum post outlining details about the runtime, so we'll want to add that as a link. There also *might* be a post for minor runtime bumps (i.e., from 3000 to 3001), but not always. So we'll always want to check that out and add a link here if there is one.

You can go to the forum https://forum.moonbeam.network/ and usually searching for 3100 or whatever the runtime version is will lead you to the post with the runtime details